### DISCUSSION TITLE
PredictPose -> Concat “No method overload found for the given arguments” error

### DISCUSSION THREAD

**Discussion by ZacariasBarreto** [bonsai-rx/bonsai#1821] [Q&A] [answered]
Hi,

I'm trying to join a video from my files with what is being captured by a camera.
When I concat videos from a FileCapture and a CameraCapture node (using a WithLatestFrom node), my project works fine, but when I put a PredictPose node between FileCapture and WithLatestFrom, I get the “No method overload found for the given arguments” error in the Concat node. 
The project (including the PredictPose node) works if I don’t use the Concat node.
Do you have any ideas of what I could do to fix this?

Thanks in advance,
zbx

![erro concat](https://github.com/bonsai-rx/bonsai/assets/103583565/42642d6d-c23f-495b-a963-b487a90177ca)

**Comment by bruno-f-cruz** [comment]
The output of predict pose is not an image, it's a object with pose related information. A function that "concatenates" an image with a pose doesn't exist. Concatenate(dsp) works on top of IplImages or Mat objects only.

Out it helps,
Bruno

**Reply by ZacariasBarreto** [reply]
Thank you for the quick answer!

I am thinking about saving what the camera captures as a video and the video of the file playing with the labels, joining them afterwards.
Is there a way in Bonsai for me to get the video with the labels then? I mean, to save what I see when I double click PredictPose while the project is running.

Best,
zbx

**Reply by bruno-f-cruz** [reply, marked as answer]
The easiest way to do this currently is to use the ScreenCapture operator
